tests/helper.rs

```diff
@@ -0,0 +1,38 @@
+use testcontainers_modules::{
+    postgres::Postgres,
+    testcontainers::{self, runners::AsyncRunner, ImageExt},
+};
+pub struct TestDb {
+    pub connection_string: String,
+    _container: testcontainers::ContainerAsync<Postgres>,
+}
+
+impl TestDb {
+    pub async fn new() -> Self {
+        let image = Postgres::default().with_tag("17.5"); // fixed tag
+        let container = image.start().await.unwrap();
```

Comment:
This could be run as a single command instruction
```suggestion
        let postgres = Postgres::default()
            .with_tag("17")
            .start()
            .await
            .expect("Failed to start Postgres instance.");
```
Prefer using major versions tag only